ca-rotation-script/script/manual-CA-Rotation.sh

```diff
@@ -17,8 +17,8 @@ mkdir new-credentials -p
 mkdir new-manifests -p
 if [[ $1 == "true" ]] 
 then
-  docker cp ${KUBE_CONTROLLER_MANAGER}:/etc/kubernetes/. backup/
-  docker cp ${KUBE_CONTROLLER_MANAGER}:/var/lib/kubelet/config.yaml ./kubelet-config-backup.yaml
+  docker cp "${KUBE_CONTROLLER_MANAGER}":/etc/kubernetes/. backup/
```

Comment:
About line 1 (github doesn't allow commenting it directly): didn't shellcheck complain about missing shebang? Usually it fails to parse scripts correctly if shebang is missing.

Please add `#!/usr/bin/env bash` as line 1.

Reply:
nope